src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/IISUrlRewriteRule.cs

```diff
@@ -13,16 +13,19 @@ public class IISUrlRewriteRule : IRule
         public UrlMatch InitialMatch { get; }
         public IList<Condition> Conditions { get; }
         public UrlAction Action { get; }
+        public bool Global { get; }
 
         public IISUrlRewriteRule(string name,
             UrlMatch initialMatch,
             IList<Condition> conditions,
-            UrlAction action)
+            UrlAction action,
+            bool? global = null)
```

Comment:
they are not equivalent. your suggestion would require that all pre-existing methods be updated to pass an explicit value. that may be desirable but i was going for least surface area change. regarding `IsGlobal` vs `Global`, i happen to share your preference. having said that, i was following the lead established elsewhere: https://github.com/aspnet/BasicMiddleware/blob/623333a919c34e63ae98a276564e03f31f9cb1cf/src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/UrlRewriteRuleBuilder.cs#L18.